src/drawing/DrawingSVGGroup.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import * as uuid from 'uuid';
+
+export type Props = React.SVGAttributes<SVGGElement> & {
+    children?: React.ReactNode;
+    onMount?: (uuid: string) => void;
+};
+
+export type DrawingSVGGroup = SVGGElement;
+
+export function DrawingSVGGroup(props: Props, ref: React.Ref<DrawingSVGGroup>): JSX.Element {
+    const { children, onMount = noop, ...rest } = props;
+    const uuidRef = React.useRef<string>(uuid.v4());
+
+    React.useEffect(() => {
+        onMount(uuidRef.current);
+    }, [onMount]);
```

Comment:
We could possibly create a custom hook to encapsulate this logic since it's used in each annotation type. Thoughts?

Reply:
Do we have similar logic elsewhere that would benefit from a reusable hook? If so, it seems like a good change.

Reply:
Add add it in a follow up PR